dapr/proto/scheduler/v1/scheduler.proto

```diff
@@ -100,6 +109,10 @@ message WatchJobsRequestInitial {
 
   // actor_types is the optional list of actor types to watch for.
   repeated string actor_types = 3;
+
+  // accept_job_types is the list of job types which the client is willing can
```

Comment:
nit grammar, but maybe good to add like this is for watching for triggered job types. its a bit clearer for future developers

```suggestion
  // accept_job_types is the list of job types which the client will watch for and accept upon triggering
```